src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -32,5 +32,20 @@ export const classes = [
   'Procedure',
 ];
 
+export const dataTypes = [
```

Comment:
Are we using the correct naming convention here?

Reply:
I have updated it to `DATA_TYPES`.